docs/development.md

```diff
@@ -39,13 +39,16 @@ In addition, the command will also *minify* the JavaScript files to reduce the s
 
 #### Starting the dev server
 
-Run the following command:
+Run the following command and wait until the task exits with a `BUILD SUCCESSFUL`:
 
 ```sh
 ./gradlew appengineRun
 ```
+Run the command below instead if you want the console output to be visible: 
+```sh
+./gradlew appengineRun -Pdisable_daemon
+```
 
-Wait until the task exits with a `BUILD SUCCESSFUL`.
 The dev server URL will be `http://localhost:8888` as specified in `build.gradle`.
 
 #### Stopping the dev server
```

Comment:
Apologies; I just realised this part about stopping the dev server using `appengineStop` is not applicable for the new command. When console output is visible, the way to stop it would be to press <kb>Ctrl + C</kbd>. Why don't you suggest a way to rearrange the instructions? :)